src/docs/framework-integration/vue.md

```diff
@@ -77,9 +78,6 @@ cd stencil-library
 npm install
 # of if you are using yarn
 yarn install
-cd ..
-lerna bootstrap
-# or if you are using other monorepo tools, initialize symlinks
 ```
 
 #### Creating a Vue Component Library
```

Comment:
So this was a section I tried to avoid a bit because I don't know Vue or any Vue toolchains. I did, just run through this whole thing end-to-end and made some changes that align more with Lerna's more recent suggestions. That cut down on some of this stuff. 

Angular doesn't use `lerna create` because we use the CLI to generate libs and apps, Vue doesn't appear to have something like that (yet?).  I'll check out React next, but I think I remember finding the same about them.

We don't need `lerna bootsrap` anymore either (I removed that in [#d313dc7](https://github.com/ionic-team/stencil-site/pull/965/commits/d313dc7c03dfc35b64b8bf50e47f6dc540d1ff01)) because Lerna now recommends leveraging workspaces: https://lerna.js.org/docs/getting-started#package-dependency-management. I made some updates for that as well.